src/test/java/core/CreateItemTest.java

```diff
@@ -5,38 +5,48 @@
 import org.jenkinsci.test.acceptance.po.FreeStyleJob;
 import org.junit.Test;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
 
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThat;
 
 public class CreateItemTest extends AbstractJUnitTest {
+
+    private static final String JOB_NAME = "asdf";
+    private static final String NAME_FIELD = "name";
+    private static final String JOB_CREATION_ERROR_MSG = "A job already exists";
+
+    private static final By NAME_NOT_EMPTY_MSG = By.id("itemname-required");
```

Comment:
Can  not use the `by` instead of the selenium `By`?? To not mix usses of `by` and `By`